### ISSUE TITLE
no errors returned when sensor not even connected

### ISSUE THREAD

**Issue by societyofrobots** [thewknd/VEML6040#5]
I'm using a custom ESP32 board, core 2.0.2, effectively running the example program.

I'm testing it with the sensor unconnected, but I'm NOT getting the expected ERROR message. It just acts like it's connected and returns 0's for data.
```
  if(!RGBWSensor.begin())
    Serial.println("ERROR: couldn't detect the sensor");
```

The log when it's not connected:
```
Waiting for a client connection to notify...
[   635][W][Wire.cpp:204] begin(): Bus already started in Master Mode.
Vishay VEML6040 RGBW color sensor auto mode example
CCT: Correlated color temperature in ⸮K
AL: Ambient light in lux
```

**Comment by Rayluc3** [comment]
I've had the same problem on a custom ESP32 board, core 2.0.0. You can resolve this problem just by adding "bool sensorExists = false;" in the src/veml6040.cpp line 39. Hope it will help you.

**Comment by thewknd** [comment]
Should be solved (b0c31b8d12d769da47a695cbb5cdea08748e87fa). Thanks for the report.
